Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -3129,5 +3388,41 @@ public int SelectedRowIndex
     /// </summary>
     [Parameter] public RenderFragment<FilterColumnContext<TItem>> FilterMenuTemplate { get; set; }
 
+    /// <summary>
+    /// Whether the DataGrid will be in batch edit mode. This will make it so every change will only be saved when <see cref="Save"/> is called.
+    /// </summary>
+    [Parameter] public bool BatchEdit { get; set; }
+
+    /// <summary>
+    /// Cancelable event before batch edit is saved.
+    /// </summary>
+    [Parameter] public EventCallback<BatchSavingEventArgs<TItem>> BatchSaving { get; set; }
+
+    /// <summary>
+    /// Event called after the batch edit is saved.
+    /// </summary>
+    [Parameter] public EventCallback<BatchSavedEventArgs<TItem>> BatchSaved { get; set; }
+
+    /// <summary>
+    /// Event called after a batch change is made.
+    /// </summary>
+    [Parameter] public EventCallback<BatchChangeEventArgs<TItem>> BatchChange { get; set; }
+
+    /// <summary>
+    /// Custom handler for the cell styling.
+    /// </summary>
+    [Parameter] public Action<TItem, DataGridColumn<TItem>, DataGridCellStyling> CellStyling { get; set; }
+
+    /// <summary>
+    /// Custom handler for the selected cell styling.
+    /// </summary>
+    [Parameter] public Action<TItem, DataGridColumn<TItem>, DataGridCellStyling> CellSelectedStyling { get; set; }

```

Comment:
We already have similar API, `SelectedRowStyling`. So I would rename this to `SelectedCellStyling`.

Reply:
I kinda would argue that this way is more discoverable. But I will change.

Reply:
Possibly. But it is better to follow the same naming pattern to what we already have.